system/core/pages.php

```diff
@@ -100,6 +100,7 @@ function newPage() {
 		html+= '<tr><th><?=__('type')?></th><td><select id="newPageType" name="newPageType">' + '<?php foreach($types as $type => $datatypeName) echo '<option value="'.$type.'"'.($type=='textpage' ? 'selected="selected"' : '').'>'.$datatypeName.'</option>'; ?>' + '</select> ' + infoPageType + '</td></tr>';
 		var infoPageName = <?=json_encode(makeInfoButton('help-page-name'));?>;
 		html+= '<tr><th><?=__('pagename')?></th><td><input type="text" id="newPagename" value="<?=$pagename?>" onblur="validatePagename(this);" onkeyup="validatePagename(this); document.getElementById(\'newPageSubmit\').disabled = this.value ? false : true;"/> ' + infoPageName + '</td></tr>';
+		html+= '<tr><th><?=__('language')?></th><td><select id="newPageLanguage" name="newPageLanguage"><?=languageOptionList('', '')?></select></td></tr>';
```

Comment:
Should this maybe preselect the default or current language? Now, nothing is marked as selected, so it just defaults to the first element in the list.

Reply:
fixed